README.md

```diff
@@ -1,20 +1,262 @@
-Cliquetis
-=========
+# Cliquetis
 
 [![Build Status](https://travis-ci.org/mozilla-services/cliquetis.svg?branch=master)](https://travis-ci.org/mozilla-services/cliquetis)
 
 A JavaScript client for [Cliquet](https://github.com/mozilla-services/cliquet/).
 
-Installation
-------------
+This is work in progress, there's no code just yet.
+
+## Installation
 
     $ npm install
 
-Build
------
+## Build
 
     $ npm run dist
 
+## Usage
+
+**Caution: everything in this section is still pure fiction.**
+
+* Every operation is performed locally;
+* Synchronization with server shall be ran explicitly.
+
+### The `Cliquetis` constructor
+
+```js
+var db = new Cliquetis(options);
+```
+
+`options` is an object defining the following option values:
+
+- `remote`: The remote Cliquet server endpoint root URL (eg. `"https://server/v1"`);
+- `headers`: The default headers to pass for every HTTP request performed to the Cliquet server (eg. `{"Authorization": "Basic bWF0Og=="}`);
+- `mode`: The conflict default resolution strategy (`Cliquet.SAFE` (*server wins*), `Cliquet.FORCE` (*client wins*)).
+
+### Collection
+
+Selecting a collection is done by calling the `collection()` method, passing it the resource name:
+
+```js
+var articles = db.collection("articles");
+```
+
+This collection object has the following attributes:
+
+* **lastModified**: last synchronization timestamp, ``null`` if never sync'ed.
+
+> Synchronization timestamps are persisted in the device local storage.
```

Comment:
I would not use a quote for this then, because that's not what it is. If it's a note, maybe we could prefix with `note: ` ?